testing/php7-libpuzzle/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Developer: Frank DENIS <j at pureftpd.org>
+# Contributor: Andreas Sundqvist <andreas at forme.se>
+# Maintainer: Timofey Bychkov <boobooin@gmail.com>
+pkgname=php7-libpuzzle
+pkgver=0.11
+pkgrel=0
+pkgdesc="PHP module of the Puzzle library. Library is designed to find visually similar images."
+url="http://libpuzzle.pureftpd.org/project/libpuzzle"
+arch="all"
+license="BSD"
+depends="libpuzzle libgd"
```

Comment:
looks strange when no `libgd-dev` used in build

Reply:
Actually we don't need this dependency for php module.
There is libpuzzle as dependency, which was build with gd-dev